documentation/modules/auxiliary/scanner/http/kyocera_addressbook_credential_gather.md

```diff
@@ -0,0 +1,27 @@
+## Vulnerable Application
```

Comment:
It might be a good idea to refer to some off our other module docs and add console outputs snippets and so on.